library/src/CMakeLists.txt

```diff
@@ -61,14 +61,10 @@ if( NOT CUDA_FOUND )
     find_package( rocsolver REQUIRED CONFIG PATHS /opt/rocm /opt/rocm/rocsolver /usr/local/rocsolver )
   endif( )
 
-  target_compile_definitions( hipsolver PRIVATE __HIP_PLATFORM_HCC__ )
-
-  target_link_libraries( hipsolver PRIVATE roc::rocblas roc::rocsolver )
+  target_link_libraries( hipsolver PRIVATE roc::rocblas roc::rocsolver hip::host )
```

Comment:
This matches hipBLAS, but I think hipBLAS is probably not handling this correctly. These should probably be `PUBLIC`. Don't worry about it for now, though. The change you have made is minimal and safe. We can investigate and address that issue separately at a later date (it's relatively minor, but requires some investigation and testing).

Reply:
Yeah, at some point I may have to ask you to review the cmake system for hipSOLVER and hopefully clean it up like you've done for rocSOLVER.